From 2b36a3b461f33d4bf01d7158250b914b4d396ca6 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 23 Jun 2017 10:37:17 +0800
Subject: [PATCH 2012/5242] MLK-15145 ARM64: dts: Correct the system counter
 freqency for i.mx8mq

commit  1f64b906d09cab51c2af439538d780bf1ff0afd9 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX8MQ, the system counter's clock frequency should be 8333333Hz.
We use 25MHz OSC, so the freqency is 25/3.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 273c24b..a242cb9 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -76,7 +76,7 @@
 			     <GIC_PPI 14 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>, /* Physical Non-Secure */
 			     <GIC_PPI 11 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>, /* Virtual */
 			     <GIC_PPI 10 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>; /* Hypervisor */
-		clock-frequency = <8000000>;
+		clock-frequency = <8333333>;
 		interrupt-parent = <&gic>;
 	};
 
-- 
1.7.9.5

